core/commands/keystore.go

```diff
@@ -34,6 +38,7 @@ publish'.
 		"list":   keyListCmd,
 		"rename": keyRenameCmd,
 		"rm":     keyRmCmd,
+		"rotate": keyRotateCmd,
```

Comment:
If we add it here, it'll show up in the global command list. To make it not show up, we'd need to "link it in" to the commands tree in cmd/ipfs/ipfs.go (_copying_ the keystore command first). Honestly, as long as we forbid calling this (make sure that works), it's probably not worth it.